installer/create-cluster-eks.sh

```diff
@@ -62,6 +62,40 @@ eksctl create nodegroup \
   --managed \
   --node-labels=dedicated=StrimziKafka
 
+# Attach EFS policy to Cloudflow roles
+cat <<EOF >describe-file-systems-policy.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": "elasticfilesystem:DescribeFileSystems",
+      "Resource": "*"
+    }
+  ]
+}
+EOF
+
+ROLE_NAME_KAFKA=$(aws eks describe-nodegroup --nodegroup-name kafka-pool-0 --cluster-name "$CLUSTER_NAME" | jq -r '.nodegroup.nodeRole')
+ROLE_NAME_STANDARD=$(aws eks describe-cluster --nodegroup-name standard-workers --cluster-name "$CLUSTER_NAME" | jq -r '.nodegroup.nodeRole')
```

Comment:
@eschizoid this command does not work, check [here](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html) for the proper parameters. 
One option is to use what I proposed in another comment:
```
$ aws eks list-nodegroups --cluster-name stavros31
{
    "nodegroups": [
        "kafka-pool-0",
        "standard-workers"
    ]
}

$ aws eks describe-nodegroup --nodegroup-name kafka-pool-0 --cluster-name stavros31 | jq  '.nodegroup.nodeRole'
"arn:aws:iam::...................:role/eksctl-stavros31-nodegroup-kafka-NodeInstanceRole-2UUW0W6ABKKF 
```


I am using btw `$ aws --version
aws-cli/1.17.9 Python/3.6.8 Linux/5.3.0-26-generic botocore/1.14.9.`

Reply:
Regarding Azure support I discussed it with my team and we are planning to publish a Cloudflow roadmap shortly and the cluster integration is part of it. Let's take it from there when we publish the details for this feature since it affects testing as well. 
@eschizoid could you also rebase/squash your commits to one in order to have one msg for the git history before merging?